snscrape/modules/telegram.py

```diff
@@ -143,6 +143,14 @@ def get_items(self):
 				raise snscrape.base.ScraperException(f'Got status code {r.status_code}')
 			soup = bs4.BeautifulSoup(r.text, 'lxml')
 
+    @staticmethod
+    def get_post_text(post) -> str:
```

Comment:
Should be `message`, not `post`, to avoid confusion with the variable in `_soup_to_items`. This should also not be public API. So `_get_message_text(message)`.